modin/pandas/test/test_io.py

```diff
@@ -1105,52 +1143,69 @@ def test_read_csv_wrong_path(self):
         )
 
     @pytest.mark.skipif(
-        StorageFormat.get() == "Hdk",
-        reason="to_csv is not implemented with HDK storage format yet - issue #3082",
+        Engine.get() == "Ray" and sys.platform.startswith("win"),
+        reason="https://github.com/modin-project/modin/issues/6171",
     )
-    @pytest.mark.parametrize("header", [False, True])
+    @pytest.mark.parametrize("extension", [None, "csv", "csv.gz"])
+    @pytest.mark.parametrize("sep", [" "])
+    @pytest.mark.parametrize("header", [False, True, "sfx-"])
     @pytest.mark.parametrize("mode", ["w", "wb+"])
+    @pytest.mark.parametrize("idx_name", [None, "Index"])
+    @pytest.mark.parametrize("index", [True, False, "New index"])
+    @pytest.mark.parametrize("index_label", [None, False, "New index"])
+    @pytest.mark.parametrize("columns", [None, ["col1", "col3", "col5"]])
```

Comment:
It looks like you have greatly increased the time of the test. We have an explicit limitation for these tests on Windows. Try increasing `timeout-minutes:` from 30 to 45 minutes.

Reply:
Done. I wonder why it takes so long on Ray? On HDK this test case takes ~9 sec in my env.

Reply:
I do not know, to find out the reason, it will be necessary to profile the tests.